dashboard/app/views/home/_privacy.md

```diff
@@ -23,49 +22,49 @@ Our Privacy Policy is fueled by our commitment to the following Privacy Principl
 Gender | Optional | This information is only used in aggregate, to measure the percentage of students that are male or female or to measure how males or females react to different computer science challenges, to track our progress towards reducing the gender gap in computer science.
 Race | Optional | Students ages 13 and over have an option to indicate their race. For students under age 13 we do not ask individual race, but we ask the teacher to optionally estimate the racial distribution of the entire classroom. <br><br>This information is only used in aggregate, to measure the percentage of students from underrepresented minorities and their aggregate reaction to computer science challenges, in order to track our progress toward improving diversity in computer science.
-Progress in the course <br>1- Date/Time each stage is tried<br>2- Number of tries to solve a puzzle, and whether it was solved successfully or optimally<br>3- Information on how the student solved the puzzle including time to completion and whether they used hints<br>4- The code that the student submitted<br>5- Student-provided answers to simple assessments (e.g. multiple-choice questions) | Mandatory (automatic) | This information is displayed to students and their teachers to see their progress in a course, to see the code they’ve created, and to identify topics they need help with. It also lets students pick up where they left off if they log out and login later. See [example progress report](https://code.org/files/student-progress.pdf), and see section below on ["technical information"](https://code.org/privacy/#techinfo) for additional details.<br><br>This data also helps Code.org improve the course effectiveness. For example, if a puzzle is too hard, Code.org may take action (like [providing better hints](https://code.org/hints)) to improve the learning process.
+Progress in the course <br>1- Date/Time each stage is tried<br>2- Number of tries to solve a puzzle, and whether it was solved successfully or optimally<br>3- Information on how the student solved the puzzle including time to completion and whether they used hints<br>4- The code that the student submitted<br>5- Student-provided answers to simple assessments (e.g. multiple-choice questions) | Mandatory (automatic) | This information is displayed to students and their teachers to see their progress in a course, to see the code they’ve created, and to identify topics they need help with. It also lets students pick up where they left off if they log out and login later. See [example progress report](http://code.org/files/student-progress.pdf), and see section below on [“technical information”](#techinfo) for additional details.<br><br>This data also helps Code.org improve the course effectiveness. For example, if a puzzle is too hard, Code.org may take action (like [providing better hints](http://code.org/hints)) to improve the learning process.
```

Comment:
Also do we want to update these two code.org links to `https` both in this file and pegasus file?